package.json

```diff
@@ -32,10 +30,12 @@
   },
   "devDependencies": {
     "anywhere": "1.5.0",
-    "icon-font-buildr": "1.3.4",
-    "js-yaml": "3.14.0",
     "pug": "3.0.0",
-    "simple-icons": "2.3.0",
-    "svgo": "1.3.2"
+    "punycode": "2.1.1",
+    "simple-icons": "4.1.0",
```

Comment:
I thought that a release wouldn't be published until the whole font build system was in order and that the next release would be in accordance with simple-icons version, but yes, maybe it's more comfortable go step by step from old ones to the new version :+1:

Reply:
I agree with your first statement, I would just try to make sure there is no accidental release at v4 to early.

I don't think it is necessary to retro-actively create a release for every simple-icons version, but I do think it could be worthwhile to create at least one for the last v2 and v3 releases.